src/main/java/aztech/modern_industrialization/compat/dashloader/DashMesh.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Azercoco & Technici4n
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package aztech.modern_industrialization.compat.dashloader;
+
+import aztech.modern_industrialization.mixin.MeshImplAccessor;
+import io.activej.serializer.annotations.Deserialize;
+import io.activej.serializer.annotations.Serialize;
+import net.fabricmc.fabric.api.renderer.v1.mesh.Mesh;
+import net.fabricmc.fabric.impl.client.indigo.renderer.mesh.MeshImpl;
+
+public class DashMesh {
+    @Serialize(order = 0)
+    public final int[] data;
+
+    @SuppressWarnings("unused")
+    public DashMesh(@Deserialize("data") int[] data) {
+        this.data = data;
+    }
+
+    public DashMesh(Mesh mesh) {
+        data = ((MeshImpl) mesh).data();
+    }
+
+    public Mesh toUndash() {
+        return MeshImplAccessor.create(data);
+    }
+}
```

Comment:
Imo this should go into dashloader's API, instead of offloading internals access to other mods.